src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.subsystems.intake;
+
+public class IntakeConstants {
+  public static final int k_indexerID = 15;
+  public static final double k_indexerSpeed = 0.8;
+  public static final double k_indexerZero = 0.0;
+  public static final int k_smartCurrentLimit = 40;
+
+  public static final double k_waitTime = 0.075;
```

Comment:
Wait time for what?  Could be more descriptive, or have a comment giving its purpose (i.e. k_retractTime // for backing the note away from the shooter).